make/migrations/postgresql/0004_1.8.0_schema.up.sql

```diff
@@ -13,6 +13,25 @@ CREATE TABLE robot (
 
 CREATE TRIGGER robot_update_time_at_modtime BEFORE UPDATE ON robot FOR EACH ROW EXECUTE PROCEDURE update_update_time_at_column();
 
+CREATE TABLE oidc_user (
+ id SERIAL NOT NULL,
+ user_id int NOT NULL,
```

Comment:
Since there's a func `dao.AddOIDCUser`
Let's make the user_id a foreign key for consistency.

Reply:
remove AddOIDCUser